tf2_gnn/layers/gnn.py

```diff
@@ -58,13 +58,13 @@ def get_default_hyperparameters(cls, mp_style: Optional[str] = None) -> Dict[str
             "initial_node_representation_activation": "tanh",
             "dense_intermediate_layer_activation": "tanh",
             "num_layers": 4,
-            "dense_every_num_layers": 2,
             "residual_every_num_layers": 2,
+            "dense_every_num_layers": 32,
+            "global_exchange_every_num_layers": 32,
             "use_inter_layer_layernorm": False,
             "hidden_dim": 16,
             "layer_input_dropout_rate": 0.0,
             "global_exchange_mode": "gru",  # One of "mean", "mlp", "gru"
-            "global_exchange_every_num_layers": 2,
```

Comment:
Is this just one more small change, unrelated to the PR comments?

Reply:
These were not meant to committed; reverted in last iteration.